bin/scheduler.ml

```diff
@@ -62,6 +74,7 @@ module Web = struct
       Server.respond_string ~status:`OK ~headers ~body ()
     | `GET, ["pool"; pool; "worker"; worker; "metrics"] -> get_metrics ~sched ~pool ~worker ~source:`Agent
     | `GET, ["pool"; pool; "worker"; worker; "host-metrics"] -> get_metrics ~sched ~pool ~worker ~source:`Host
+    | `GET, ["pool"; pool; "worker"; worker; extra] -> get_metrics ~sched ~pool ~worker ~source:(`Extra extra)
```

Comment:
What does this API look like for extra requests? For example I want to request the `clarke` stats, would I make a request "/pool/linux-ppc64/worker/ppc-worker-1/clarke" and that gives me a collection of reported clarke stats?

Reply:
Yep that's the idea, finally managed to get on a linux machine and take it for a spin. Seems to be working, so for example `/pool/linux-x86_64/worker/my-host/clarke` would give Clarke metrics. The main ones we want:

```
#HELP Clarke_meter_intensity Current carbon intensity in gCO2/kWh
#TYPE Clarke_meter_intensity gauge
Clarke_meter_intensity 0.000000
#HELP Clarke_meter_watts Current power usage measured in watts
#TYPE Clarke_meter_watts gauge
Clarke_meter_watts 100.000000
```